crates/xilem_html/src/lib.rs

```diff
@@ -10,10 +10,11 @@ use wasm_bindgen::JsCast;
 mod app;
 mod class;
 mod context;
-mod diff;
+pub mod diff;
 mod element;
 mod event;
 mod one_of;
+pub mod vecmap;
```

Comment:
What if you just moved the doc tests for this into a regular set of tests and then we wouldn't expose this API?

Reply:
Hmm yes I thought about it, but IMO I think that having the examples directly in the rust function docs, with tests verifying that they work is nice, without possibly dedicated duplicated(?) tests as the exposed functionality isn't really "critical" internals (rather stuff that may have use for other crates as well) IMO (there are ways btw. to [avoid](https://users.rust-lang.org/t/doctest-of-private-functions/60805/5) having to expose this by default with a feature flag specifically for testing, might be worth exploring). But I can of course avoid exposing and extract all this into dedicated tests, anyone else agrees here?